InvenTree/templates/account/base.html

```diff
@@ -89,30 +89,5 @@
 <!-- general JS -->
 {% include "third_party_js.html" %}
 
-<script type='text/javascript' src="{% static 'script/inventree/inventree.js' %}"></script>
-<script type='text/javascript' src="{% i18n_static 'notification.js' %}"></script>
-
-
-<script type='text/javascript'>
-
-$(document).ready(function () {
-    // notifications
-    {% if messages %}
-    {% for message in messages %}
-    showAlertOrCache(
```

Comment:
We will be missing sign-out / going into maintenance alters without this. That is a heavy regression in my eye.

Reply:
Might have to look into splitting the javascript files then, into once which are "safe" for non-logged-in-users to view.

Reply:
@matmair LMK what you think of the new updates. I have put the messages back in, but removed any ability for the user to see sensitive JS files which require login.